usecase/usecase-webapi-xwalk-tests/steps/SetCookie/step.js

```diff
@@ -0,0 +1,8 @@
+var step = '<font class="fontSize">'
+         + '<p>Test Purpose: </p>'
+         + '<p>Verifies HTTP response header "Set-Cookie".</p>'
+         + '<p>Test steps: </p>'
+         + '<p>Check the "Set-Cookie" value.</p>'
+         + '<p>Expected Result: </p>'
+         + '<p>Test passes if it should get the response header value expect "Set-Cookie", the value of "Set-Cookie" should be blank(null).</p>'
```

Comment:
Refer to http://www.w3.org/TR/XMLHttpRequest/#the-getresponseheader()-method:
<code>
  If header is a case-insensitive match for Set-Cookie or Set-Cookie2, return null.
</code>